### PR TITLE
Improve the build-process

### PR DESCRIPTION
**Pull request by larsgrefer**

### CODE REVIEW

**Comment by SecUpwN** [comment]
@larsgrefer, would like to merge this, but it appears to have merge conflicts.

File: app/build.gradle [hunk 1]
```diff
@@ -39,68 +47,28 @@ android {
...
+
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_7
         targetCompatibility JavaVersion.VERSION_1_7
     }
-    productFlavors {
-    }
 
     lintOptions {
-        disable 'MissingTranslation'
```
Comment (agilob) [inline app/build.gradle]: You can safely disable it, weblate takes care of missing translations.

Comment (larsgrefer) [inline app/build.gradle]: I prefer to keep this as warning, so it appears in the results and statistics

**Comment by SecUpwN** [comment]
Thanks for fixing, @larsgrefer! May I merge this now? :+1:

**Comment by larsgrefer** [comment]
in my opinion, yes

**Pull request merged**